core/src/type_info.rs

```diff
@@ -0,0 +1,306 @@
+use crate::{
+    bson_type_info::BsonTypeInfo, col_metadata::MongoColMetadata, conn::MongoConnection,
+    err::Result, stmt::MongoStatement, Error,
+};
+use bson::Bson;
+use odbc_sys::{Nullability, SqlDataType};
+
+use lazy_static::lazy_static;
+
+// pub static ref DATA_TYPES: Vec<BsonTypeInfo> = vec![
+// const DATA_TYPES: [&BsonTypeInfo; 21] = [
+//     &BsonTypeInfo::BOOL,                // SqlDataType(-7)
+//     &BsonTypeInfo::LONG,                // SqlDataType(-5)
+//     &BsonTypeInfo::BINDATA,             // SqlDataType(-2)
+//     &BsonTypeInfo::ARRAY,               // SqlDataType(0)
+//     &BsonTypeInfo::BSON,                // SqlDataType(0)
+//     &BsonTypeInfo::DBPOINTER,           //SqlDataType(0)
+//     &BsonTypeInfo::DECIMAL,             // SqlDataType(0)
+//     &BsonTypeInfo::JAVASCRIPT,          //SqlDataType(0)
+//     &BsonTypeInfo::JAVASCRIPTWITHSCOPE, // SqlDataType(0)
+//     &BsonTypeInfo::MAXKEY,              // SqlDataType(0)
+//     &BsonTypeInfo::MINKEY,              // SqlDataType(0)
+//     &BsonTypeInfo::NULL,                // SqlDataType(0)
+//     &BsonTypeInfo::OBJECT,              // SqlDataType(0)
+//     &BsonTypeInfo::OBJECTID,            // SqlDataType(0)
+//     &BsonTypeInfo::SYMBOL,              //SqlDataType(0)
+//     &BsonTypeInfo::TIMESTAMP,           // SqlDataType(0)
+//     &BsonTypeInfo::UNDEFINED,           // SqlDataType(0)
+//     &BsonTypeInfo::INT,                 // SqlDataType(4)
+//     &BsonTypeInfo::DOUBLE,              // SqlDataType(8)
+//     &BsonTypeInfo::STRING,              // SqlDataType(12)
+//     &BsonTypeInfo::DATE,                // SqlDataType(93)
+// ];
```

Comment:
```suggestion
```
old dead code?

Reply:
yes -- thought I had gotten all of it, and should have double checked before the re-request. apologies for the mess!